doc/source/getting_started.rst

```diff
@@ -28,180 +29,72 @@ To test that everything works properly, open up IPython::
-
-    * Canopy >= 1.0 (Mac, Linux)
-
-    * Anaconda (Mac)
-
-    * Debian / Ubuntu standard system Python + Scipy stack
-
-Anaconda
-^^^^^^^^
-
-Note for developers. To make Anaconda working with our test-suite a few
-manual adjustments might be necessary. This may require
-manually adjusting the python interpreter invoked by the nosetests and
-the sphinx-build 'binaries' (http://goo.gl/Atqh26).
-Tested on a recent MacBook Pro running Mac OS X 10.8 and Mac OS X 10.9
-
-multi-threading
-^^^^^^^^^^^^^^^
-
-For optimal performance we recommend using numpy / scipy with the multi-threaded
-ATLAS, gotoblas2, or intel MKL. For example, the Enthought Canopy and the Anaconda distributions
-ship with tested MKL-compiled numpy / scipy versions. Depending on the use case and your system
-this may speed up operations by a factor greater than 10.
-
-pylab
-^^^^^
-
-Although all of the examples in this documentation are in the style
-of the standard Python interpreter, the use of IPython with the pylab option
-is highly recommended. In addition, for the setups listed above we would
-strongly recommend to use the QT matplotlib backend for fast and correct rendering::
-
-    ipython --pylab qt
-
-
-On Linux, for example, QT is the only matplotlib backend for which 3D rendering
-will work correctly. On Mac OS X for other backends certain matplotlib functions
-might not work as expected.
-
-IPython notebooks
-^^^^^^^^^^^^^^^^^
-
-To take full advantage of MNE-Python's visualization capacities in combination
-with IPython notebooks and inline displaying, please explicitly add the
-following magic method invocation to your notebook or configure your notebook
-runtime accordingly.
-
-    %pylab inline
-
-If you use another Python setup and you encounter some difficulties please
-report them on the MNE mailing list or on github to get assistance.
-
-
-.. _inside_martinos:
-
-Inside the Martinos Center
---------------------------
-
-For people within the MGH/MIT/HMS Martinos Center mne is available on the network.
-
-In a terminal do::
-
-    setenv PATH /usr/pubsw/packages/python/epd/bin:${PATH}
-
-If you use Bash replace the previous instruction with::
-
-    export PATH=/usr/pubsw/packages/python/epd/bin:${PATH}
-
-Then start the python interpreter with:
-
-    ipython
-
-Then type::
+For example, here we use a simplistic pipeline to go from raw data to brain
+source time courses in under 30 lines of code:
 
+    >>> # import the necesary packages and functions
+    >>> from os import path as op
     >>> import mne
-
-If you get a new prompt with no error messages, you should be good to go.
-Start with the :ref:`examples-index`.
+    >>> from mne.datasets.sample import data_path
+
+    >>> # structural information (made in Freesurfer and MNE-C)
+    >>> subjects_dir = op.join(data_path(), 'subjects')
+    >>> bem_dir = op.join(subjects_dir, 'sample', 'bem')
+    >>> mri = op.join(data_path(), 'MEG', 'sample', 'sample_audvis_raw-trans.fif')
```

Comment:
can you add a comment that the trans file is the only file that requires manual coregistration? eg. using mne_analyze or mne.coreg module?